frontend/lib/src/baseconsts.ts

```diff
@@ -14,18 +14,11 @@
  * limitations under the License.
  */
 
-/**
- * When in dev mode, this is the port used to connect to the web server that is
- * serving the current page (i.e. the actual web page server, not the API
- * server, which in dev are actually different servers.)
- */
-export const WWW_PORT_DEV = 3000
```

Comment:
`WWW_PORT_DEV` seems to have been unused previously?

Reply:
Yea. I couldn't find this const in any search.